language-support/hs/bindings/examples/group-chat/BUILD.bazel

```diff
@@ -8,10 +8,12 @@ load("@os_info//:os_info.bzl", "is_windows")
 daml_compile(
     name = "GroupChat",
     srcs = glob(["daml/GroupChat.daml"]),
+    enable_scenarios = True,  # TODO: https://github.com/digital-asset/daml/issues/11316
```

Comment:
while they’re called examples the HS bindings are only used internally so can be post 2.0